community/contribution_guide/release.md

```diff
@@ -385,86 +403,75 @@ Checklist for reference:
 
 [ ] No compiled archives bundled in source archive.
 
-More detail checklist please refer:
-https://cwiki.apache.org/confluence/display/INCUBATOR/Incubator+Release+Checklist
-
 
---
+— 
 
 Best Regards
-Chao Tian
+Naijie Liu
+
 ```
 
 #### Voting close
 ```bash
-[VOTE] Release Apache SeaTunnel 2.3.3
-
-Hi SeaTunnel Community,
-
-Thanks, everyone, I will close this vote thread and the results will be tallied.
-
-Best wishes!
-Chao Tian
-#### 归票
-[RESULT] [VOTE] Release Apache SeaTunnel 2.3.3
+[RESULT] [VOTE] Release Apache SeaTunnel 2.3.8
 
 Hi SeaTunnel community,
 
 This vote now closes since 72 hours have passed.
 
 The vote PASSES with
 
-3 (+1 binding) votes from the IPMC,
```

Comment:
Can we have below format

There are 6 approving votes, 4 of which are binding:
- Jia Fan (binding)
- Guangdong Liu (binding)
- hailin0 (binding)
- David (binding)
- Mohammad Arshad
- User name 6


2 (+0 no opinion) vote
- User name 7
- User name 8

There are no disapproving votes

The vote passes with 4 binding +1 votes and 2 non-binding +1 votes